README.md

```diff
@@ -77,3 +77,20 @@ Clone the repo and look through (CONTRIBUTING.md)[CONTRIBUTING.md]
 % cd starfish
 ```
 
+## citing starfish
+
+to cite starfish, please use: 
+
+Axelrod S, Carr AJ, Freeman J, Ganguli D, Long B, Tung T, and others. 
+Starfish: Open Source Image Based Transcriptomics and Proteomics Tools, 2018-, 
+http://github.com/spacetx/starfish [Online; accessed <date>].
+
+Here’s an example of a BibTeX entry:
+
+@misc{,
+author = {Shannon Axelrod, Ambrose J Carr, Jeremy Freeman, Deep Ganguli, Brian Long, Tony Tung, and others},
+title = {{Starfish}: Open Source Image Based Transcriptomics and Proteomics Tools},
+year = {2018--},
```

Comment:
two dashes?

Reply:
Copied from `scipy`. Maybe that means open-ended?